chronograf/server/mux.go

```diff
@@ -131,11 +128,6 @@ func NewMux(opts MuxOpts, service Service) http.Handler {
 		)
 	}
 
-	if opts.PprofEnabled {
-		// add profiling routes
-		router.GET("/debug/pprof/:thing", http.DefaultServeMux.ServeHTTP)
-	}
```

Comment:
AFAIK this mux isn't actually connected to anything, so this is just cleanup/consolidation